server/fleet/software.go

```diff
@@ -170,3 +185,66 @@ func (uhsdbr *UpdateHostSoftwareDBResult) CurrInstalled() []Software {
 
 	return r
 }
+
+// ParseSoftwareLastOpenedAtRowValue attempts to parse the last_opened_at
+// software column value. If the value is empty or if the parsed time is
+// 0 it returns (time.Time{}, nil).
+func ParseSoftwareLastOpenedAtRowValue(value string) (time.Time, error) {
+	// We don't fail if only the last_opened_at cannot be parsed.
```

Comment:
Nit, but just so it doesn't make things confusing - we do fail if it cannot be parsed (by `strconv.ParseFloat`), we don't if it's empty.